R/sim_anneal_helpers.R

```diff
@@ -144,11 +144,31 @@ random_real_neighbor <- function(current, hist_values, pset, retain = 1,
 
 encode_set_backwards <- function(x, pset, ...) {
   pset <- pset[pset$id %in% names(x), ]
-  new_vals <- purrr::map2(pset$object, x, dials::encode_unit, direction = "backward")
+  new_vals <- mapply(encode_unit_backwards, pset$object, x,
```

Comment:
Transitioned to `mapply()` so that we don't get purrr's `Error in index i, caused by [call we can't control]` rethrown error formatting.